src/marten/db/query/sql/raw_statement_node.cr

```diff
@@ -0,0 +1,70 @@
+require "./concerns/sanitizer"
+
+module Marten
+  module DB
+    module Query
+      module SQL
+        class RawStatementNode < PredicateNode
```

Comment:
Could we rename this class `RawPredicateNode` in order to improve consistency (considering that the parent class is named `PredicatedNode`)? 🙏